klib/mbedtls_conf.h

```diff
@@ -0,0 +1,73 @@
+#ifndef _RUNTIME_H_
+#include <runtime.h>
+#endif
+
+#define NULL    ((void *)0)
+
+#define CHAR_BIT    8
+
+#define INT_MAX     0x7fffffff
+#define UINT_MAX    0xffffffff
+#define SIZE_MAX    0x7fffffffffffffffll
+
+typedef signed char int8_t;
+typedef unsigned char uint8_t;
+typedef short int16_t;
+typedef unsigned short uint16_t;
+typedef int int32_t;
+typedef unsigned int uint32_t;
+typedef long int64_t;
+typedef unsigned long uint64_t;
+
+typedef unsigned long size_t;
+typedef unsigned long uintptr_t;
```

Comment:
I think it would be fine for other platforms too: as long as we are targeting 64-bit platforms only, these typedefs won't change, a long should be 64-bit on any 64-bit platform.